analyzer/uncategorized/addresses.go

```diff
@@ -42,6 +45,15 @@ func StringifyEthAddress(ethAddr []byte) (apiTypes.Address, error) {
 	return StringifyOcAddress(ocAddr)
 }
 
+func EthAddrReference(preimage string) *string {
+	if len(preimage) == 0 {
+		return nil
+	}
+
+	ref := ethCommon.HexToAddress(strings.Trim(preimage, "\\x")).String()
```

Comment:
Here we're hardcoding a peculiarity of postgres (`bytea` is returned serialized as hex, with "\\x" prepended), in a module completely separate from sql.

Let's fix the encoding on the sql side instead:
`select encode(address_data, 'hex') from ...`
https://stackoverflow.com/questions/44813386/mysqls-hex-and-unhex-equivalent-in-postgres

This Trim() can go away then.

Reply:
aaah the json object making us not able to have byte arrays